internal/cmd/functional-benchmarker/main.go

```diff
@@ -83,9 +81,15 @@ func RunBenchmark(artifactDir string, options config.Options) (*stats.ResourceMe
 	}
 	statistics := gatherStatistics(runner, options.Sample, options.MsgSize, startTime, endTime)
 
+	go func() { fetchContainerLogs(runner, artifactDir) }()
```

Comment:
This will run asynchronously with no synchronization and will most likely continue running after RunBenchmark() returns. Why not just call `fetchContainerLogs()` synchronously?

Reply:
I was trying to speed up the test runs and this change is the same that was part of the 5.7 merge and does not seem to have resulted in any issues

Reply:
That's why it's called a race condition - it doesn't cause any issues, until it does :) Would it make sense to keep main() from returning until fetchContainerLogs() is done?

Reply:
Updated to remove go routine